Numsense/Russian.fs

```diff
@@ -0,0 +1,154 @@
+﻿module internal Ploeh.Numsense.Russian
+
+open Ploeh.Numsense.InternalDsl
+
+type internal Gender =
+    | Masculine
+    | Feminine
+    | Neuter
+
+let rec internal toRussianImp gender x =
+
+    let simplify gender prefix factor x =
+        let remainder = x % factor
+        if remainder = 0
+        then prefix
+        else sprintf "%s %s" prefix (toRussianImp gender (remainder))
+
+    let impUnderThousand gender x =
+        let simplify = simplify gender
+        match x with

```

Comment:
Oh, I've overlooked this one. Maybe it would be good to enable `treat warnings as errors` compiler setting.

I've handled the default case now, by raising an `InvalidOperationException` with some informative message.